src/v2/coins/abstractUtxoCoin.js

```diff
@@ -261,6 +261,9 @@ class AbstractUtxoCoin extends BaseCoin {
             // the address was found, but not on the wallet, which simply means it's external
             debug('Address %s presumed external', currentAddress);
             return _.extend({}, currentOutput, { external: true });
+          } else if (e.message.includes('address validation failure: invalid chain') && currentAddress === txParams.changeAddress) {
+            // expect to see this error when passing in a changeAddress with no chain or index
```

Comment:
Passing in a **custom** changeAddress - subtle but important distinction (we should have chain and index information for normal change addresses)